test/disyuntor.test.js

```diff
@@ -296,8 +296,51 @@ describe('disyuntor', function () {
         },
       ], done);
     });
+  });
 
+  describe('should throttle', function() {
+    var sut;
+    beforeEach(function () {
+      sut = disyuntor({
+        name: 'test.func',
+        timeout: 10,
+        maxFailures: 1,
+        cooldown: '10ms',
+        action: 'throttle',
+        throttlePercent: 50
+      }, (callback) => {
+        callback(new Error('failure'));
+      });
+    });
 
+    it('should allow requests through and fail others', function(done) {
```

Comment:
This will fail around 1 time a year 0.001 assuming it is run every day as probability of failure.... I'm too picky, I don't like non-deterministic tests, but I think we might be fine